src/tests/ftest/container/open.yaml

```diff
@@ -1,6 +1,7 @@
 hosts:
   test_servers:
     - server-A
+timeout: 340
```

Comment:
This timeout seems very large based upon these test results:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3803/2/artifact/Functional/job-2020-11-06T01.31-f355574-container-open/results.html
```
container-gooduuid-goodpoh-hosts-pool-server_config-7b37   PASS	   31.24
container-gooduuid-badpoh-hosts-pool-server_config-add7    PASS	   28.70
container-baduuid-goodpoh-hosts-pool-server_config-0a92    PASS	   28.74
container-baduuid-badpoh-hosts-pool-server_config-6848	   CANCEL  22.30
```

Reply:
I usually get the test time from the output near the end like below.
Total test time: 174s

Then I multiply it by 1.5 and use that value for timeout.

Reply:
Yes, but the timeout is per test variant.  Using your '1.5' rule the timeout should be set using `max(31.24, 28.70, 28.74, 22,30) * 1.5 = 48`

Reply:
Thanks. It looks like these tests are executed with fairly consistent amount of time, so fixing the timeout value isn't urgent. I ran the whole set of tests in CI, so I'd rather fix this in the next PR.